fintrack/fintrack/urls.py

```diff
@@ -19,5 +19,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('api/', include('transactions.urls')),
+    path('api/transactions/', include('transactions.urls')),
```

Comment:
are we not doing versions?

Reply:
Sir, not yet as Implementing User registration and authentication first.